subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5.h

```diff
@@ -44,6 +47,14 @@
 #error "Unsupported SoC."
 #endif
 
+#include <hal/nrf_rtc.h>
+#include <hal/nrf_timer.h>
+
+#if defined(CONFIG_BT_CTLR_LE_ENC) || defined(CONFIG_BT_CTLR_BROADCAST_ISO_ENC)
```

Comment:
Since `defined(CONFIG_BT_CTLR_LE_ENC) || defined(CONFIG_BT_CTLR_BROADCAST_ISO_ENC)` appears so many places, we could consider having a hidden helper Kconfig option to check against instead

Reply:
any suggested name?

Reply:
Nothing that comes to mind that wouldn't get confused with other values :)